### ISSUE TITLE
Category selector is incompatible with .NET 8

### ISSUE THREAD

**Issue by Tim-Schmelter** [Geta/geta-optimizely-categories#31]
We upgraded our projects to .NET 8 but needed to go back to .NET 6 since the geta catgeory selector is not compatible and we get errors in the console wherever we use the category selector in the cms:

![geta_errors](https://github.com/Geta/geta-optimizely-categories/assets/66748193/95fbd995-d920-4c91-aa07-42a30545fdf0)

**Comment by evest** [comment]
Strange, I have 1.1.1 running fine on .NET 8.

**Comment by Tim-Schmelter** [comment]
I just noticed something: in my screenshot you can see that it seems that the 1.0.0 script was loaded even if we were using the 1.1.1 version(now we rolled back to 1.0.0). What we were doing wrong then?

**Comment by evest** [comment]
Yeah; I wondered as well, assumed the screenshot was from after rolling back.

It is also returned with the wrong content-type in your screenshot, which is even more strange.

Seem like a cache issue, what does the modules\_protected\Geta.Optimizely.Categories\Geta.Optimizely.Categories.zip look like? Is it 1.1.1 inside after you have upgraded?

Is the new zipfile deployed, or does it pick the old one?

Does it work locally in a dev env.?

**Comment by GeekInTheNorth** [comment]
I'm not observing this issue with Geta Categories 1.1.3
